lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
@BenRussert You can add breakpoints inside the dev tools panel. Take a look at: https://developers.google.com/web/tools/chrome-devtools/javascript/breakpoints